client/components/theme/index.jsx

```diff
@@ -162,12 +141,6 @@ export class Theme extends Component {
 		this.props.onStyleVariationClick?.( this.props.theme.id, this.props.index, variation );
 	};
 
-	isBeginnerTheme() {
```

Comment:
We currently do not have any themes that use this functionality, so I'm leaving this implementation out for the meantime.